crud/select/filters.lua

```diff
@@ -491,6 +510,22 @@ local function lt_boolean_strict(lhs, rhs)
     return (not lhs) and rhs
 end
 
+local function lt_uuid_nullable(lhs, rhs)
+    if lhs == nil and rhs ~= nil then
+        return true
+    elseif rhs == nil then
+        return false
+    end
+    return lhs:str() < rhs
```

Comment:
How should I understand why `lhs:str()` us compared with `rhs`? It should be obvious by arguments names. 
For example, `lt_uuid_nullable(operand, value)`.

Reply:
My suggestion is `tostring(lhs) < tostring(rhs)`

Reply:
Renamed rhs values